wormchain/x/wormhole/client/cli/genesis.go

```diff
@@ -427,7 +428,7 @@ func CmdSignAddress() *cobra.Command {
 				return err
 			}
```

Comment:
could you explain the context and the need for this code? Isn't this accomplished through the guardian admin command already? I'm not excited about reading in the guardian key as an environment variable and having an additional code path and operator instructions for handling the guardian key.

Reply:
I was using it for testing.  Guardians will be advised to use the admin command that was added.

I'll update to make this more clear.